lib/store_web/views/error_view.ex

```diff
@@ -7,13 +7,6 @@ defmodule StoreWeb.ErrorView do
   #   "Internal Server Error"
   # end
 
-  def render("404.html", assigns) do
-    render(StoreWeb.ErrorView, "404.html",
```

Comment:
This was the problem, render "404.html" for the loop in the same template "404.html"

Reply:
ErrorView.render("404.html") was calling ErrorView.render("404.html") which is infinite recursion....